import React from "react";
import Layout from "./LayoutPage"; // Import the Layout component

function ContactPage() {
  return (
    <Layout>
       <div className="container mx-auto py-8">
      <h1 className="text-4xl font-bold text-center mb-6">DenGen Bioinformatics Pipeline</h1>

      <div className="bg-white p-6 rounded-lg shadow-md">
        <h2 className="text-2xl font-semibold text-[#003865] mb-4">DenGen Bioinformatics Pipeline</h2>
        <p className="text-gray-600 mb-6">To generate a population-wide variant frequency resource for Denmark, we analyzed whole-genome sequencing (WGS) data from 2,916 individuals obtained through routine clinical pipelines. Sequencing was performed using paired-end reads on an Illumina NovaSeq6000 platform with the Illumina DNA PCR-free (tagmentation) kit, achieving an average read depth of at least 52X per sample.</p>
 <p className="text-gray-600 mb-6"> High-quality variant discovery and filtering were performed using a standardized in-house bioinformatics pipeline optimized for both single-nucleotide variants (SNVs) and structural variants (SVs). SNVs were identified using GATK’s HaplotypeCaller, providing sensitive and accurate detection. Structural variants, including copy-number variants (CNVs), were detected using a consensus approach that integrates multiple SV callers, CNVnator, Delly, Lumpy, and Manta to enhance robustness and mitigate the limitations of individual tools. </p>
       

<h2 className="text-2xl font-semibold text-[#003865] mb-4">Overview</h2>
       
 <p className="text-gray-600 mb-6"> The DenGen pipeline follows a systematic multi-step process to transform raw sequencing data into a reliable set of annotated variants: </p>


       
        {/* Pipeline Steps Section */}
        <div className="space-y-6">
          <div>
            <h4 className="text-lg font-semibold text-[#003865]">1. Data Preprocessing</h4>
            <p className="text-gray-600">
              Raw sequencing data generated by the <strong>Illumina NovaSeq 6000</strong> system are demultiplexed using <strong>bcl2fastq</strong>, and quality metrics are assessed to ensure each sample has sufficient reads for downstream analysis.
Subsequently, FASTQ files are trimmed with <strong>bbduk</strong> to remove adapter sequences and low-quality bases. 
            </p>
          </div>

          <div>
            <h4 className="text-lg font-semibold text-[#003865]">2. Alignment</h4>
            <p className="text-gray-600">

             Trimmed reads are aligned to the human reference genome using the <strong>Sentieon implementation of the Burrows-Wheeler Aligner (BWA)</strong>, configured for paired-end sequencing data. This step ensures accurate placement of reads within the genome.
Alignment quality is assessed using metrics such as median coverage, and only samples achieving a minimum median coverage of 30× are retained for downstream analyses. 
            </p>
          </div>

          <div>
            <h4 className="text-lg font-semibold text-[#003865]">3. Variant Calling</h4>
            <p className="text-gray-600">
              Variant detection is performed using the <strong>GATK HaplotypeCaller</strong> for SNVs, providing high sensitivity and accuracy. For structural variants, a consensus approach combining tools such as <strong>CNVnator</strong>, <strong>Delly</strong>, <strong>Lumpy</strong>, and <strong>Manta</strong> is employed to detect copy number variations (CNVs) and SVs.
            </p>
          </div>

          <div>
            <h4 className="text-lg font-semibold text-[#003865]">4. Variant Annotation</h4>
            <p className="text-gray-600">
              Once variants are called, they are annotated with information such as functional impact, allele frequency, and related clinical information using tools like <strong>VEP (Variant Effect Predictor)</strong>.
            </p>
          </div>

           {/*
  
   

          <div>
            <h4 className="text-lg font-semibold text-[#003865]">5. Quality Control and Filtering</h4>
            <p className="text-gray-600">
              After variant annotation, we perform rigorous quality control steps to ensure the validity of the variant calls. Filtering criteria are applied based on quality scores, read depth, and other metrics to remove false positives and low-confidence variants.
             
            </p>
          </div>

          <div>
            <h4 className="text-lg font-semibold text-[#003865]">6. Data Integration and Final Output</h4>
            <p className="text-gray-600">
              The final output includes a comprehensive list of high-confidence variants, ready for further analysis. These variants are stored in the DenGen database, providing researchers and clinicians with valuable insights into the genetic makeup of the Danish population.
            </p>
          </div>

          */}
        </div>
        
        {/*       
        <div className="mt-6">
          <h3 className="text-xl font-semibold text-[#003865] mb-4">Access the Data</h3>
          <p className="text-gray-600 mb-4">
            The results of our bioinformatics pipeline are stored in the DenGen database, which is accessible to researchers and clinicians through our public and secure data access portals. If you're interested in using the data, please refer to our <a href="/data-access" className="text-[#003865] hover:underline">Data Access Page</a> for more details.
          </p>
        </div>
         */}
      </div>
    </div>
    </Layout>
  );
}
export default ContactPage;



