% This file was produced by plot-vcfstats, the command line was:
%   plot-vcfstats -p /mnt/dengen/results/outdir/71cofverm-103815072020-Normal_Blood_noinfo-WGS_v1_IlluminaDNAPCRFree_X-211217_A01411_BHHJ3KDSX2-RHGM_LABKA_WGS-WGS03506 /mnt/dengen/results/stats/71cofverm-103815072020-Normal_Blood_noinfo-WGS_v1_IlluminaDNAPCRFree_X-211217_A01411_BHHJ3KDSX2-RHGM_LABKA_WGS-WGS03506_snv_germline_raw.haplotype_caller.stats
%
% Edit as necessary and recreate the PDF by running
%   pdflatex summary.tex
%

% Slides style and dimensions
%
\nonstopmode
\documentclass[17pt]{memoir}
\setstocksize{19cm}{25.4cm}
\settrimmedsize{\stockheight}{\stockwidth}{*}
\settrims{0pt}{0pt}
\setlrmarginsandblock{1cm}{*}{*}
\setulmarginsandblock{1.5cm}{*}{*}
\setheadfoot{1mm}{1cm}
\setlength{\parskip}{0pt}
\setheaderspaces{*}{1mm}{*}
\setmarginnotes{1mm}{1mm}{1mm}
\checkandfixthelayout[fixed]
\usepackage{charter}   % font
\pagestyle{plain}
\makeevenfoot{plain}{}{}{\thepage}
\makeoddfoot{plain}{}{}{\thepage}
\usepackage{graphicx}

% For colored tables. If xcolor.sty is not available on your system,
% download xcolor.sty.gz LaTeX class style from
%   http://www.ukern.de/tex/xcolor.html
% Unpack and install system-wide or place elsewhere and make available by
% setting the TEXINPUTS environment variable (note the colon)
%   export TEXINPUTS=some/dir:
% The list of the recognised path can be obtained by running `kpsepath tex`
%
\usepackage{multirow}
\setlength{\tabcolsep}{0.6em}
\renewcommand{\arraystretch}{1.2}
        
\usepackage[table]{xcolor}
\definecolor{hcol1}{rgb}{1,0.6,0}
\definecolor{hcol2}{rgb}{1,0.68,0.2}
\definecolor{row1}{rgb}{1,0.88,0.7}
\definecolor{row2}{rgb}{1,0.92,0.8}    % #FFEBCC
\setlength{\arrayrulewidth}{1.5pt}

% Slide headings
\newcommand*{\head}[1]{{\Large\centerline{#1}\vskip0.5em}}

% Slide definition
\newcommand*{\hslide}[2]{%
        \head{#1}%
        \begin{vplace}[0.5]\centerline{#2}\end{vplace}\newpage}
\newcommand{\pdf}[2]{\IfFileExists{#2}{\includegraphics[#1]{#2}}{}}


% The actual slides
\begin{document}
        
% Table with summary numbers
%
\hslide{Summary Numbers}{
\begin{minipage}{\textwidth}\centering
\small \rowcolors*{3}{row2}{row1} \arrayrulecolor{black}
\begin{tabular}{l | r r r | r r | r | r}
\multicolumn{1}{>{\columncolor{hcol1}}l|}{}
& \multicolumn{3}{>{\columncolor{hcol1}}c|}{SNPs}
& \multicolumn{2}{>{\columncolor{hcol1}}c|}{indels}
& \multicolumn{1}{>{\columncolor{hcol1}}c|}{MNPs}
& \multicolumn{1}{>{\columncolor{hcol1}}c}{others}  \\
%
\multicolumn{1}{>{\columncolor{hcol2}}l|}{Callset}
& \multicolumn{1}{>{\columncolor{hcol2}}c}{n}
& \multicolumn{1}{>{\columncolor{hcol2}}c }{ts/tv}
& \multicolumn{1}{>{\columncolor{hcol2}}c|}{\footnotesize(1st ALT)}
& \multicolumn{1}{>{\columncolor{hcol2}}c}{n}
& \multicolumn{1}{>{\columncolor{hcol2}}c}{frm$^*$}
& \multicolumn{1}{>{\columncolor{hcol2}}c|}{}
& \multicolumn{1}{>{\columncolor{hcol2}}c}{} \\ \hline
         71cof & 4,164,440 & 1.92 & 1.92 & 968,828 & -- & 0 & 0 \\ 
%
        \multicolumn{8}{r}{$^*$ frameshift ratio: out/(out+in)} \\
        \end{tabular}
        \\ \vspace{1em}
        \begin{tabular}{l | r r r | r r}
        \multicolumn{1}{>{\columncolor{hcol1}}l|}{}
        & \multicolumn{3}{>{\columncolor{hcol1}}c|}{singletons {\footnotesize(AC=1)}}
        & \multicolumn{2}{>{\columncolor{hcol1}}c}{multiallelic}  \\
        %
        \multicolumn{1}{>{\columncolor{hcol2}}l|}{Callset}
        & \multicolumn{1}{>{\columncolor{hcol2}}c}{SNPs}
        & \multicolumn{1}{>{\columncolor{hcol2}}c}{ts/tv}
        & \multicolumn{1}{>{\columncolor{hcol2}}c}{indels}
        & \multicolumn{1}{>{\columncolor{hcol2}}c}{sites}
        & \multicolumn{1}{>{\columncolor{hcol2}}c}{SNPs} \\ \hline
     71cof & 62.9\% & 1.88 & 67.9\% & 106,058 &2,053 \\ 
 \end{tabular}
\vspace{2em}
\begin{itemize}[-]
\setlength{\itemsep}{0pt}
        \item 71cof .. \texttt{\footnotesize /ngc/projects2/gm/data/archive/2022/variants/snv/71cofverm-103815072020-Normal\_B\\\hskip2em lood\_noinfo-WGS\_v1\_IlluminaDNAPCRFree\_X-211217\_A01411\_BHHJ3KDSX2-RHGM\_LABKA\_WGS-\\\hskip2em WGS03506\_snv\_germline\_raw.haplotype\_caller.vcf.gz}
\end{itemize}\end{minipage}}

        
% Ts/Tv by sample
%
\hslide{Ts/Tv by sample}{\vbox{\centerline{\includegraphics[height=7cm]{{tstv_by_sample.0}.png}}}}
        
% Hets vs non-ref Homs by sample
%
\hslide{Hets vs non-ref Homs by sample}{\vbox{\centerline{\includegraphics[height=7cm]{{hets_by_sample.0}.png}}}}
        
% Singletons by sample {\normalsize(hets and homs)}
%
\hslide{Singletons by sample {\normalsize(hets and homs)}}{\vbox{\centerline{\includegraphics[height=7cm]{{singletons_by_sample.0}.png}}}}
        
% Average depth by sample
%
\hslide{Average depth by sample}{\vbox{\centerline{\includegraphics[height=7cm]{{dp_by_sample.0}.png}}}}
        
% Number of SNPs by sample
%
\hslide{Number of SNPs by sample}{\vbox{\centerline{\includegraphics[height=7cm]{{snps_by_sample.0}.png}}}}
        
% Number of indels by sample
%
\hslide{Number of indels by sample}{\vbox{\centerline{\includegraphics[height=7cm]{{indels_by_sample.0}.png}}}}
        
% Ts/Tv stratified by QUAL
%
\hslide{Ts/Tv stratified by QUAL}{\includegraphics[width=15cm]{{tstv_by_qual.0}.png}}
        
% Indel distribution
%
\hslide{Indel distribution}{\includegraphics[width=15cm]{{indels.0}.png}}
        
% Fraction of alternate indel allele
%
\hslide{Fraction of alternate indel allele}{\vbox{\centerline{\includegraphics[height=7cm]{{indel_vaf.0}.png}}}}
        
% Depth distribution
%
\hslide{Depth distribution}{\includegraphics[width=15cm]{{depth.0}.png}}
        
% Number of HETs by AF
%
\hslide{Number of HETs by AF}{\includegraphics[width=15cm]{{hwe.0}.png}}
        
% Substitution types
%
\hslide{Substitution types}{\includegraphics[width=15cm]{{substitutions.0}.png}}
        
\end{document}
